### DISCUSSION TITLE
bug: copy selected text to clipboard from Browse Code contains non-breaking space

### DISCUSSION THREAD

**Discussion by networkhermit** [community/community#51661] [Mobile]
## steps to reproduce

1. use github mobile on android

2. navigate to https://github.com/cli/cli/blob/trunk/go.mod

3. select the first line and tap the copy button

## expected behavior

the copied text contains exactly the line in source code

## unexpected behavior

the copied text contains non-breaking space (utf-8 c2a0):

```text
module github.com/cli/cli/v2
```

```text
$ base64 | base64 -d | cat -v
module github.com/cli/cli/v2
moduleM-BM- github.com/cli/cli/v2
```

```text
$ echo module github.com/cli/cli/v2 | xxd
00000000: 6d6f 6475 6c65 c2a0 6769 7468 7562 2e63  module..github.c
00000010: 6f6d 2f63 6c69 2f63 6c69 2f76 320a       om/cli/cli/v2.
```

## outcome

the copy code to clipboard functionality is effectively unusable, for example copying a single line bash to termux would fail because of the non-breaking space

**Comment by iskae** [comment]
Hello @networkhermit 👋 

Thanks so much for reaching out ❤️ 

I will be creating an internal issue for this and we will try to fix this as soon as possible 👍 

Please stay tuned for the feature releases 🙇 

Cheers!
